packages/components/package.json

```diff
@@ -44,7 +44,6 @@
     "@kaizen/typography": "^2.3.10",
     "@kaizen/draft-tag": "^3.4.16",
     "@kaizen/brand": "^1.5.8",
-    "@kaizen/draft-modal": "^10.6.19",
```

Comment:
Btw why aren't these alphabetical 😛

Reply:
It's a secret alphabet... where "t" comes before "b"... Now you're in the inner circle 😈

Reply:
Couldn't unsee it so I've fixed that now 😆